EpiAware/src/latent-models.jl

```diff
@@ -26,3 +21,90 @@ end
     end
     return rw, (; σ_RW, rw_init)
 end
+
+# Define the Priors type alias
+const Priors = Union{Distribution, Vector{<:Distribution}, Product}
+
+struct AR <: AbstractLatentModel
+    """A distribution representing the prior distribution of the damping factors."""
+    damp_prior::Priors
+
+    """A distribution representing the prior distribution of the variance."""
+    var_prior::Distribution
+
+    """A distribution representing the prior distribution of the initial values."""
+    init_prior::Priors
+
+    """
+    The order of the AR process, determined by the length of `damp_prior`.
+    """
+    p::Int
+
+    function AR(damp_prior::Priors, var_prior::Distribution, init_prior::Priors)
```

Comment:
I think making at least one constructors with kwargs might be nice?

Reply:
Can you dummy out what you mean?

Reply:
See #133